examples/03_connectivity/plot_data_driven_parcellations.py

```diff
@@ -188,6 +198,100 @@
 kmeans_labels_img.to_filename('kmeans_parcellation.nii.gz')
 
 ##################################################################
-# Finally show them
+# Brain parcellations with ReNA Clustering
+# ----------------------------------------
+# The spatial constraints are implemented inside the ReNA object.
```

Comment:
Here say what the benefit of ReNA: it is very fast for a large number of parcels.